### ISSUE TITLE
twitter connection not working, but no errors

### ISSUE THREAD

**Issue by marchawkins** [idno/Twitter#33]
I can't get my posts to publish to twitter. I've tried revoking access and re-authenticating, deleting and creating a new twitter app, logging out and back in of everything. I can't get it to work. The known dashboard seems to indicate something is connected to twitter, but it doesn't show any accounts (see attached). Any other ideas I can try? Thanks
<img width="553" alt="screen shot 2016-01-08 at 10 37 13 am" src="https://cloud.githubusercontent.com/assets/3503548/12201759/186b8d4e-b5f4-11e5-9600-bf1663a7cae4.png">

**Comment by herbsmn** [comment]
I'm having a similar problem, but I was getting errors in my nginx log folder. 

It said 'error' => 'Read-only application cannot POST.'

I tried reconnecting my app to twitter and this is what I saw:

![screenshot6](https://cloud.githubusercontent.com/assets/8562898/16602422/77185bfa-4303-11e6-873a-d0baa9be787a.png)

So I logged into dev.twitter.com/apps and created a new App specifically making sure that it had read and write priveledges.

I then made sure to revoke access to any other Known related apps here: https://twitter.com/settings/applications

When I tried to connect my Known site after doing these things it connected properly, asking for write access too.

I hope this helps!
